spec/spec_helper.rb

```diff
@@ -46,7 +46,7 @@ def new_connection
   when 'mysql'
     password_argument = (PASSWORD.empty?) ? "" : "-p#{PASSWORD}"
     Kernel.system %{ mysql -h 127.0.0.1 -u #{CURRENT_USER} #{password_argument} -e "DROP DATABASE IF EXISTS #{DATABASE}" }
-    Kernel.system %{ mysql -h 127.0.0.1 -u #{CURRENT_USER} #{password_argument} -e "CREATE DATABASE #{DATABASE} CHARSET utf8" }
+    Kernel.system %{ mysql -h 127.0.0.1 -u #{CURRENT_USER} #{password_argument} -e "CREATE DATABASE #{DATABASE} CHARSET utf8mb4 COLLATE utf8mb4_general_ci" }
```

Comment:
does this mean we are replacing testing utf8/mysql with something more esoteric?

if so, shouldn't we test both?